doc/release/1.13.0-notes.rst

```diff
@@ -29,9 +32,8 @@ Future Changes
 Compatibility notes
 ===================
 
-
 Error type changes
-~~~~~~~~~~~~~~~~~~
```

Comment:
Note that we have a regression in #8845 here, which was merged at just the wrong time